### DISCUSSION TITLE
Role and localization of P-gp in lungs

### DISCUSSION THREAD

**Discussion by Nathalie06** [Open-Systems-Pharmacology/Forum#776] [Questions & Problems]
Hi, 

I am trying to model an intranasal administration in PK-sim, I used different absorption routes for the dose. For around 20% of the dose I used "User defined administration type" with lung as target organ and blood cells as  target compartment. My compound is a P-gp substrate and I have seen P-gp is expressed in lungs in PK-sim.
My question is, does P-gp influence the entry of my compound in the systemic circulation using this particular route ? and if yes, how ?

Thanks in advance
Best,
Nathalie

**Comment by prvmalik** [comment]
Hi Nathalie

Proteins, enzymes, transporters etc. are only functional in PKSim when you have added them into your 'Individual' building block and parameterized a relevant ADME process in your 'Compound' building block. So unless you have added P-gp, it will not be affecting your compound in the model.

By default, passive movement of the drug between the blood cells and the plasma is determined by the 'Blood-to-Plasma' ratio, the permeability of the drug across the blood cell membrane and the surface area of the blood cells.

Paul

**Comment by Nathalie06** [comment]
Hi Paul, 

Thanks for your answer. I added P-gp in my 'Individual' and I added in my compound a Vmax and Km for P-gp. 
As I used P-gp as an efflux transporter, the compound should be transporter from intracellular space to interstitial space or lumen. So in my case I am not sure of the impact or the no-impact.
I made DDI prediction with a P-gp inductor and the effect is an Cmax ratio > 1 (I don't understand why because my dose is split in oral dose, infusion dose and user defined with lung. So for the oral dose I think the Cmax ratio should be lower than 1, there is no impact on the infusion, so I am wondering if my user defined administration could be responsible for this Cmax ratio > 1 ?).

Thanks
Nathalie

**Comment by prvmalik** [comment]
PK-Sim does not allow transporters in blood cells (for now). So there will be no effect on the movement of the drug from blood cell space to plasma space.

I recommend using the "P-gp like" transporter type, rather than the "Efflux" transporter type for P-gp. Changing this may solve your problem.

As you search for the source of this phenomena, be aware that P-gp is present in many organs in the PK-Sim model and drives efflux in key areas, such as into the intestinal lumen, into the bile and into the urine. Inducing P-gp usually lowers absorption and increases clearance.

**Comment by Nathalie06** [comment]
Thanks Paul !
